manager/src/main/java/com/redhat/service/bridge/manager/api/user/BridgesAPI.java

```diff
@@ -52,26 +61,57 @@
     @Inject
     JsonWebToken jwt;
 
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeListResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403")
+    })
+    @Operation(summary = "Get the list of ingresses", description = "Get the list of ingresses for the authenticated user.")
     @GET
     public Response getBridges(@Valid @BeanParam QueryInfo queryInfo) {
         System.out.println(jwt.getSubject());
         return Response.ok(ListResponse.fill(bridgesService
                 .getBridges(identityResolver.resolve(jwt), queryInfo), new BridgeListResponse(), bridgesService::toResponse)).build();
     }
 
+    @APIResponses(value = {
+            @APIResponse(description = "Accepted.", responseCode = "202",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403")
+    })
+    @Operation(summary = "Create an ingress", description = "Create an ingress for the authenticated user.")
     @POST
     public Response createBridge(BridgeRequest bridgeRequest) {
         Bridge bridge = bridgesService.createBridge(identityResolver.resolve(jwt), bridgeRequest);
         return Response.status(Response.Status.CREATED).entity(bridgesService.toResponse(bridge)).build();
     }
 
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403")
```

Comment:
Please add HTTP 404 if the Bridge doesn't exist.

Reply:
With the JSON Accept header, as the response is :
```
{
  "kind": "Error",
  "id": "4",
  "href": "/api/v1/errors/4",
  "code": "OPENBRIDGE-4",
  "reason": "Bridge with id 'qqq/q' for customer '1608dc1f-4139-49c5-ae22-afe73ed807d7' does not exist"
}
```

Reply:
done